Tasks/ExtractFilesV1/task.json

```diff
@@ -58,6 +58,14 @@
             "required": true,
             "defaultValue": "false",
             "helpMarkDown": "Select this option to overwrite existing files in the destination directory."
+        },
+        {
+            "name": "pathToSevenZipTool",
+            "type": "string",
+            "defaultValue": "",
+            "label": "Path to 7z utility",
+            "required": false,
+            "helpMarkDown": "You can specify custom path to 7z utility"
```

Comment:
Could you please add an example to the help information?

Reply:
Sure, added